projects/ngx-smart-banner/src/lib/ngx-smart-banner.component.html

```diff
@@ -1,3 +1,9 @@
+<head>
+    <link
```

Comment:
Unfortunately, this will end up as invalid HTML when someone uses the package. And in packages you preferably don't use these kind of dependencies.  

My reccomendation is to create a star icon with the SVG of font awesome. You can use fill="currentColor" to style the svg using the color css: 

**StarComponent html**
```Html
<ng-container *ngIf="solid; else regular">
<svg class="star-rating" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><path fill="currentColor" d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"/></svg>
</ng-container>
<ng-template #regular>
<svg class="star-rating" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><path fill="currentColor" d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"/></svg>
</ng-template>
```